helm/opendistro-es/templates/elasticsearch/es-client-deploy.yaml

```diff
@@ -56,6 +56,7 @@ spec:
       affinity:
 {{ toYaml . | indent 8 }}
     {{- end }}
+{{ if .Values.elasticsearch.sysctl.enabled }}
```

Comment:
Hi @kfox1111,
Could you add "- " in front of the "if" (all files) to keep consistency in chopping whitespace?
Also, please move this line below "initContainers" like in other files. Thanks.

Reply:
woops. missed the -. Will fix.

Do you really want an initContainer section with no values in it? That seems less clean?

Reply:
Thanks for the response.
Let's keep the consistency here, I think it is fine to have that section left as is.
We can add more steps later if we need to.

Reply:
Hi @kfox1111,
Thanks for commits.
Please also make same changes to other .yaml files as well for the "- ".
helm/opendistro-es/templates/elasticsearch/es-data-sts.yaml
helm/opendistro-es/templates/elasticsearch/es-master-sts.yaml
Thanks.